airbyte-integrations/connectors/destination-weaviate/destination_weaviate/destination.py

```diff
@@ -0,0 +1,94 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+import string
+from typing import Any, Iterable, Mapping
+import random
+
+from airbyte_cdk import AirbyteLogger
+import logging
+from airbyte_cdk.destinations import Destination
+from airbyte_cdk.models import AirbyteConnectionStatus, AirbyteMessage, ConfiguredAirbyteCatalog, Status, Type
+
+from .client import Client
+
+
+def get_schema(configured_catalog: ConfiguredAirbyteCatalog) -> Mapping[str, Mapping[str, str]]:
```

Comment:
I would prefer if utility methods are transferred to a utility class

Reply:
ok will move them to a uttils.py file

Reply:
done, please verify and resolve